mock/centralized_server.go

```diff
@@ -9,17 +9,17 @@ import (
 	cid "github.com/ipfs/go-cid"
 	ds "github.com/ipfs/go-datastore"
 	dssync "github.com/ipfs/go-datastore/sync"
-	peer "github.com/libp2p/go-libp2p-peer"
-	pstore "github.com/libp2p/go-libp2p-peerstore"
-	"github.com/libp2p/go-testutil"
+
+	"github.com/libp2p/go-libp2p-core/peer"
```

Comment:
@raulk IMO, we should be preferring the type aliases for types like this (as I expect user code will use those).